### ISSUE TITLE
add option to load-balance model-fitting

### ISSUE THREAD

**Issue by copernican** [nikita-moor/ldatuning#5]
Fitting a model via Gibbs sampling tends to scale in the number of topics `k`, with larger values of `k` taking longer to fit. Because the user may be interested in tuning `k` over a large range, the models fit for smaller values of `k` will tend to take less time than those for larger values of `k`. The implementation of `parLapply` may not be as efficient as a load-balanced approach for this scenario. The parallelization may be better implemented using `clusterApplyLB`, which in general does a better job of managing utilization when the distributed tasks take variable lengths of time.

Accordingly, would it be appropriate to modify `FindTopicsNumber` to take a logical argument `lb`, with parallelization similar to the following? If so, happy to submit a PR.

```r
  lda.fun <- function(x) {
    topicmodels::LDA(dtm, k = x, method = method, control = control)
  }
  if (lb) {
    models <- do.call(
      parallel::clusterApplyLB, 
      args = list(x = topics, fun = lda.fun)
    )
  } else {
    models <- do.call(
      parallel::parLapply, 
      args = list(X = topics, fun = lda.fun)
    )
  }
```

**Comment by nikita-moor** [comment]
There is a more simple and straightforward balancing method: calculate the longest decomposition (big value of `k`) first and move to smaller `k`. Also, issue #4 should be taken into account.

Nevertheless, original goals of the project - optimal distribution metrics - were not achieved. All my attempts to contact with the authors of (Deveaud2014) were fruitless, so I'm not interesting in future development of this project and just keep it working. You can send me a patch and I'll apply it, or I can give you commit rights if you are going to make significant development.

**Comment by titaniumtroop** [comment]
The solution mentioned above — to calculate the larger values of k first — was implemented in #12 last year.
